Symfony/CS/Fixer/Contrib/PhpUnitConstructFixer.php

```diff
@@ -26,24 +44,44 @@ public function fix(\SplFileInfo $file, $content)
     {
         $tokens = Tokens::fromCode($content);
 
-        for ($index = 0, $limit = $tokens->count(); $index < $limit; ++$index) {
-            $skipToIndex = $this->fixAssertNotSame($tokens, $index);
+        if ($this->configuration['assertNotEquals']) {
```

Comment:
The alternative will be calling method by variable, sth like `$this->$callback($tokens, $index, $methodToBeFixed)`, not sure if it will be more readable